cmd/stack_removefromrepo.go

```diff
@@ -121,25 +121,14 @@ The updated repository index file is created in  ~/.appsody/stacks/dev.local dir
 					if err != nil {
 						return errors.Errorf("Error checking status of %s", localIndexFile)
 					}
-					if exists && useLocalCache {
-						log.Debug.Log(localIndexFile, " exists in the appsody directory and use-local-cache is true")
+					if exists {
+						log.Debug.Log(localIndexFile, " exists in the appsody directory")
 					} else {
-						log.Info.Log("Repository index file not found - unable to remove stack")
-						return nil
+						return errors.Errorf("Repository index file not found - unable to remove stack")
```

Comment:
After speaking to @groeges - it would be better if we kept this as it was, e.g. printing an Info in this case, maybe a Debug in other cases and returning nil from the function.